R/db-odbc-mssql.R

```diff
@@ -100,3 +100,22 @@
   )
   DBI::dbExecute(con, sql)
 }
+
+#' @export
+`db_save_query.Microsoft SQL Server` <- function(con,
+                                                 sql,
+                                                 name,
+                                                 temporary = TRUE,
+                                                 ...)
+{
+  # check that name has prefixed '##' if temporary
+  if(temporary && substr(name, 1, 1) != "#") {
+    name <- paste0("##", name)
+    message("Created temporary table named: ", name)
+  }
+
+  tt_sql <- build_sql("select * into ", as.sql(name), " from (", sql, ") ", as.sql(name), con = con)
```

Comment:
Should `name` appear twice in this query?

Reply:
Yes, the last instance it the table alias name for the query.  I just tried it w/o it and the operation failed

Reply:
It seems slightly odd that's valid SQL, but if it works it's fine.